pricing-page-exercise/kadir/index.html

```diff
@@ -4,9 +4,98 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+    <title>Pricing for Bootstrap KCS</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
+    <div class="container_all">
+      
+      <header>
```

Comment:
The header element is not taking 100% of the width of the page so it make the navigation bar look incorrect. You should be able to fix it by setting header width to 100%